frameworks/cassandra/tests/conftest.py

```diff
@@ -3,10 +3,14 @@
 import sdk_security
 from tests import config
 
+from sdk_utils import random_string
```

Comment:
We generally prefer: `import sdk_utils` and then use `sdk_utils.random_string()` explicitly.